src/components/VersionMismatch.tsx

```diff
@@ -0,0 +1,135 @@
+import { useEffect } from 'react';
+
+import {
+  Button,
+  Icon,
+  Link,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+  useDisclosure,
+} from '@chakra-ui/react';
+import {
+  IconBrandGithub,
+  IconExclamationCircleFilled,
+  IconThumbUpFilled,
+} from '@tabler/icons-react';
+
+import useVersions from '../store/useVersions';
+
+function VersionMismatch(): JSX.Element | null {
+  const { smesherVersion, versionCheck } = useVersions();
+  const disclosure = useDisclosure();
+
+  const { onOpen } = disclosure;
+  useEffect(() => {
+    if (!versionCheck?.supported) {
+      onOpen();
+    }
+  }, [onOpen, versionCheck?.supported]);
+
+  if (!versionCheck || (versionCheck.supported && !versionCheck.hasUpdate))
+    return null;
+
+  const url = `https://smesher-alpha.spacemesh.network/v${
```

Comment:
I think that needs to be different. Will be tricky to do deployments of two things to the one location and keep consistency between them. 
Also I'm not fully understand the expected file format. Why don't we have it literally https://smesher-alpha.spacemesh.network/versions.json ? Is there any reason that you want to have it *per version* ?

Having it dynamic in that way would make it tricky to host.

Reply:
It is the link to the correct version of APP that we're hosting, not versions file.

There will be a single versions.json file, path is specified in the hard-coded constant:
https://github.com/spacemeshos/smesher-app/pull/20/files/07be940c07c97524422d1055a859b02d8ea5b91a#diff-b6ee6afc2c1dc3b1883e37483c3c82dda5bdd91499a45bbceadce75883d386f3R10-R11

The format of the file as described in the issue: a list of tuples, that contains semvers: `(MinSmesherVersion, LatestAppVersion)[]`